### PR TITLE
qca 2.3.1

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/qca.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Qca < Formula
   head "https://invent.kde.org/libraries/qca.git"
 
   stable do
-    url "https://github.com/KDE/qca/archive/v2.3.0.tar.gz"
-    sha256 "39aa18f0985d82949f4dccce04af3eb8d4b6b64e0c71785786738d38d8183b0a"
+    url "https://download.kde.org/stable/qca/2.3.1/qca-2.3.1.tar.xz"
+    sha256 "c13851109abefc4623370989fae3a745bf6b1acb3c2a13a8958539823e974e4b"
 
     # use major version for framework, instead of full version
```
Comment (SMillerDev) [inline Formula/qca.rb:10]: Has this been reported in the proper upstream repo?

Comment (yurikoles) [reply Formula/qca.rb:10]: `download.kde.org` is recommended download source for all KDE software, it's actually a load balancer powered by MirrorBrain. And it was in this release announce: https://mail.kde.org/pipermail/kde-announce-apps/2020-July/005594.html

Comment (yurikoles) [reply Formula/qca.rb:10]: Oh, sorry, wrong line. I had sent this commit [upstream](https://invent.kde.org/libraries/qca/-/merge_requests/34).

Comment (SMillerDev) [reply Formula/qca.rb:10]: Is there a way to use that as the source?

Comment (yurikoles) [reply Formula/qca.rb:10]: Its actually was already merged.

**Comment by yurikoles** [comment]
@SMillerDev I had enabled additional plugins and added corresponding dependencies.

File: Formula/qca.rb [hunk 2]
```diff
@@ -24,22 +24,26 @@ class Qca < Formula
...
     args << "-DBUILD_TESTS=OFF"
     args << "-DQCA_PLUGINS_INSTALL_DIR=#{lib}/qt5/plugins"
 
     # Disable some plugins. qca-ossl, qca-cyrus-sasl, qca-logger,
     # qca-softstore are always built.
-    args << "-DWITH_botan_PLUGIN=NO"
-    args << "-DWITH_gcrypt_PLUGIN=NO"
-    args << "-DWITH_gnupg_PLUGIN=NO"
-    args << "-DWITH_nss_PLUGIN=NO"
-    args << "-DWITH_pkcs11_PLUGIN=NO"
```
Comment (chenrui333) [inline Formula/qca.rb:42]: Since we enabled these plugins, do we want to mention in the `caveats`?

Comment (yurikoles) [reply Formula/qca.rb:42]: AFAIK `caveats` are intended to mention brew-specific issues. I don't think that adding extra functionality is the correct case.

**Comment by SMillerDev** [comment]
Are these plugins the recommended default by upstream?

**Review by SMillerDev — APPROVED** [review]
Thanks @yurikoles ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**